packages/quantic/cypress/tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "resolveJsonModule": true,
     "esModuleInterop": true
   },
-  "include": ["**/*.ts", "**/*.js"]
+  "include": ["../node_modules/cypress", "**/*.ts", "**/*.js"]
```

Comment:
Not sure why this is required ? Feels weird to have to include something from node_modules here.
Just an initial reaction, this might be the right thing to do.

Reply:
Yea this is a "fix" to something weird, because we both use Jest and Cypress in the ui-kit repo, and they both export stuff globally like `describe, it, expect, etc.` it causes conflict in the intellisense of vscode.

This is a fix suggested by Cypress to resolved these conflicts. Notice that this `tsconfig` is under the /cypress folder, so the ts intellisense resolves it's settings by considering the closest tsconfig, so by including cypress here directly, it makes sure that the globally exported stuff by cypress are considered "over" the globally exported stuff by Jest.

Long explanation just to say this fixes a small technical issue with autocomplete and stuff in the cypress tests :)